examples/cpp/02_fused_two_gemms/fused_gemm.tsv

```diff
@@ -1,12 +0,0 @@
-[M, N, K, P, kBatch]	[kTM, kTN, kTK, kTP]	cuBLAS	TiledCUDA	Ratio
```

Comment:
I prefer not to add the data file into the main codebase. It can be hosted in another repository.